keel-core/src/main/kotlin/com/netflix/spinnaker/keel/jackson/KeelApiModule.kt

```diff
@@ -63,6 +65,7 @@ object KeelApiModule : SimpleModule("Keel API") {
       setMixInAnnotations<SubnetAwareRegionSpec, SubnetAwareRegionSpecMixin>()
       setMixInAnnotations<Resource<*>, ResourceMixin>()
       setMixInAnnotations<ResourceSpec, ResourceSpecMixin>()
+      setMixInAnnotations<ArtifactSortByMethod, ArtifactSortByMethodMixin>()
```

Comment:
for my knowledge, what is this doing?

Reply:
Adding the mixin to customize Jackson serialization of the enum, so that we can use use lower-case values.